crowbar_framework/spec/controllers/crowbar_controller_spec.rb

```diff
@@ -245,10 +245,10 @@
 
   describe "proposal updates" do
     before(:each) do
-      Proposal.any_instance.stubs(:save).returns(true)
-      CrowbarService.any_instance.expects(:validate_proposal).at_least_once
-      CrowbarService.any_instance.expects(:validate_proposal_elements).returns(true).at_least_once
-      CrowbarService.any_instance.expects(:validate_proposal_after_save).at_least_once
+      allow_any_instance_of(Proposal).to receive(:save).and_return(true)
+      expect_any_instance_of(CrowbarService).to receive(:validate_proposal)
+      expect_any_instance_of(CrowbarService).to receive(:validate_proposal_elements).and_return(true)
```

Comment:
Metrics/LineLength: Line is too long. [101/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)